app/code/Deity/Store/Test/Api/StoreConfigManagerTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+declare(strict_types=1);
+
+namespace Deity\Store\Test\Api;
+
+use Magento\Integration\Model\AdminTokenServiceTest;
+use Magento\Store\Api\StoreConfigManagerTest as MagentoStoreConfigManagerTest;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+/**
+ * Class StoreConfigManagerTest
+ * @package Deity\Store\Test\Api
+ */
+class StoreConfigManagerTest extends WebapiAbstract
+{
+    /**
+     * @return string
+     */
+    private function getAccessToken(): string
+    {
+        $adminUserNameFromFixture = 'webapi_user';
+
+        $serviceInfo = [
+            'rest' => [
+                'resourcePath' => AdminTokenServiceTest::RESOURCE_PATH_ADMIN_TOKEN,
+                'httpMethod' => \Magento\Framework\Webapi\Rest\Request::HTTP_METHOD_POST,
+            ],
+        ];
+
+        $requestData = [
+            'username' => $adminUserNameFromFixture,
+            'password' => \Magento\TestFramework\Bootstrap::ADMIN_PASSWORD,
+        ];
+        $accessData = $this->_webApiCall($serviceInfo, $requestData);
+        return $accessData['token'];
+    }
+
+    /**
+     * @magentoApiDataFixture Magento/Webapi/_files/webapi_user.php
+     */
+    public function testGetStoreConfigs()
+    {
+        $serviceInfo = [
+            'rest' => [
+                'resourcePath' => MagentoStoreConfigManagerTest::RESOURCE_PATH,
+                'httpMethod'  => \Magento\Framework\Webapi\Rest\Request::HTTP_METHOD_GET,
+                'token' => $this->getAccessToken()
+            ]
+        ];
+
+        $storeViews = $this->_webApiCall($serviceInfo, []);
+        $this->assertTrue(isset($storeViews[0]['extension_attributes']));
```

Comment:
using the message parameter in assertions is helpful to understand it, especially when they fail
```suggestion
        $this->assertTrue(isset($storeViews[0]['extension_attributes']), 'Store view should have extension attributes');
```